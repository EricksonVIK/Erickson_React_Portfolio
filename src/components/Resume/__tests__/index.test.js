import React from "react";
import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import Resume from "../index.resume";

afterEach(cleanup);

describe("Resume component", () => {
  // baseline test
  it("renders", () => {
    render(<Resume />);
  });
  // snapshot test
  it("matches snapshot", () => {
    const { asFragment } = render(<Resume />);
    // assert value comparison
    expect(asFragment()).toMatchSnapshot();
  });
});
