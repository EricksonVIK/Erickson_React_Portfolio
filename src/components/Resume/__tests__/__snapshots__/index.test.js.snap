// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resume component matches snapshot 1`] = `
<DocumentFragment>
  <section>
    <h2>
      Resume
    </h2>
    <p>
      To download my resume,
      <a
        href="Erickson_Resume.pdf"
      >
        click here
      </a>
    </p>
    <h3>
      Front-End Proficiencies
    </h3>
    <ul>
      <li>
        HTML
      </li>
      <li>
        CSS
      </li>
      <li>
        JavaScript
      </li>
      <li>
        JQuery
      </li>
      <li>
        Responsive design
      </li>
      <li>
        React
      </li>
      <li>
        BootStrap, Materialize, Bulma
      </li>
    </ul>
    <h3>
      Back-End Proficiencies
    </h3>
    <ul>
      <li>
        APIs
      </li>
      <li>
        Node
      </li>
      <li>
        Express
      </li>
      <li>
        MySQL, Sequelize
      </li>
      <li>
        MongoDB, Mongoose
      </li>
      <li>
        REST
      </li>
      <li>
        GraphQL
      </li>
    </ul>
  </section>
</DocumentFragment>
`;
